pages/login.page.js

```diff
@@ -0,0 +1,33 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import BasePage from '../pages/base.page';
+
+class LoginPage extends BasePage {
+
+   //WebElements
+   get usuarioInput(){ return $('[name="username"]') }
+   get contraseñaInput(){return $('[name="password"]')}
+
+
+   //-------------------------------------------------------------------------------------------------------//
+
+   /**
+    * Escribe el usuario y la contraseña en los campos y presiona Enter
+    * @param {String} usuario 
+    * @param {String} contraseña
+    */
+   async ingresar(usuario, contraseña) {
+       await super.vaciarCampoYEnviarTexto(await this.usuarioInput, usuario);
+       await super.vaciarCampoYEnviarTexto(await this.contraseñaInput, contraseña);
+       await this.contraseñaInput.keys('Enter');
+   }
+
+   /**
+    * Obtener texto del usuario
+    */
+   async obtenerTextoUsuario() {
```

Comment:
Prefer a more "domain oriented" description for this method. Methods, besides good means for encapsulating logic / complexity, also provide opportunities for **abstracting out** knowledge about an operation, so clients (like code in spec.js files) don't have to know the details of a given implementation before using it. In this regard, a better name for this method could be **getUsername()**, as it more direclty reflects that you expect to obtain a string representation of a username than current name (**user text** does not inmediatly evoke that meaning)